.github/actions/install-gitea/install-gitea.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# ------------------------------------------------------------
+# Copyright 2023 The Radius Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------
+
+set -e
+
+GITEA_USERNAME=$1
+GITEA_EMAIL=$2
+GITEA_ACCESS_TOKEN_NAME=$3
+# GITEA_PASSWORD should be set by environment variable
+
+if [ -z "$GITEA_USERNAME" ]; then
+  echo "GITEA_USERNAME is not set. Exiting..."
+  exit 1
+fi
+
+if [ -z "$GITEA_EMAIL" ]; then
+  echo "GITEA_EMAIL is not set. Exiting..."
+  exit 1
+fi
+
+if [ -z "$GITEA_ACCESS_TOKEN_NAME" ]; then
+  echo "GITEA_ACCESS_TOKEN_NAME is not set. Exiting..."
+  exit 1
+fi
+
+# Add Gitea Helm chart repository
+helm repo add gitea-charts https://dl.gitea.io/charts/
+helm repo update
+
+# Install Gitea from Helm chart
+helm install gitea gitea-charts/gitea --version v11.0.0 --namespace gitea --create-namespace -f .github/actions/install-gitea/gitea-config.yaml
```

Comment:
Is the version expected to remain static? or would we have something similar to dependabot to update it every so often?

Reply:
For now leaving it static is okay - we do this with lots of things, even running on install (contour, dapr). This tool is only used for testing so IMO leaving it here is ok.